hummingbot/strategy/__utils__/trailing_indicators/historical_volatility.py

```diff
@@ -0,0 +1,26 @@
+from .base_trailing_indicator import BaseTrailingIndicator
```

Comment:
I am not sure if the team has set a coding standard, but I personally try not to use relative imports.